Doc/library/stdtypes.rst

```diff
@@ -3243,8 +3245,8 @@ produce new objects.
    *chars* argument is a binary sequence specifying the set of byte values to
    be removed - the name refers to the fact this method is usually used with
    ASCII characters.  If omitted or ``None``, the *chars* argument defaults
-   to removing ASCII whitespace.  The *chars* argument is not a prefix;
-   rather, all combinations of its values are stripped::
+   to removing :term:`ASCII whitespace <whitespace>`.  The *chars* argument is
```

Comment:
Likewise, better to link to https://docs.python.org/3/library/stdtypes.html#bytes.isspace?